c/meson.build

```diff
@@ -142,6 +145,29 @@ if get_option('default_library') == 'static'
 endif
 
 
+# Service API
+nns_capi_service_shared_lib = shared_library ('capi-nnstreamer-service',
```

Comment:
Discussion:
let's consider library name,
capi-ml-common
capi-nnstreamer-single > capi-ml-inference-single
capi-nnstreamer-pipeline > capi-ml-inference-pipeline
capi-nnstreamer-service > capi-ml-service

Reply:
Good! I prefer

Reply:
Good! I prefer
 - capi-ml-inference-single
 - capi-ml-inference-pipeline
 - capi-ml-service (for future updates)

Reply:
Applied!